lib/pengine/native.c

```diff
@@ -655,7 +658,12 @@ pcmk__native_output_string(pe_resource_t *rsc, const char *name, pe_node_t *node
         const char *desc = crm_element_value(rsc->xml, XML_ATTR_DESC);
 
         if (desc) {
-            pcmk__add_word(&outstr, 0, desc);
+            size = strlen("(") + strlen(desc) + strlen(")");
+            desc_with_symbols = malloc(size);
+            strcpy(desc_with_symbols, "(");
+            strcat(desc_with_symbols, desc);
+            strcat(desc_with_symbols, ")");
+            pcmk__add_word(&outstr, 0, desc_with_symbols);
```

Comment:
We have a convenience function that makes this easier like:

    desc_with_symbols = crm_strdup_printf("(%s)", desc);

It's dynamically allocated, so be sure to free it when you're done with it.

As with the XML, there are group/clone/bundle versions of this as well. I don't think they output the description at all currently, but they probably should.